src/server/controllers/socketController.js

```diff
@@ -0,0 +1,128 @@
+const { isDynamicMetadataRoute } = require('next/dist/build/analysis/get-page-static-info');
+const dbClient = require('../utils/db');
+const { ObjectId } = require('mongodb');
+
+
+const SocketController = {
+	async createRoom(req, res) {
+		try {
+			const {userId, name, isRoomChat} = req.body
+
+			if (!name || !userId) {
+				throw new Error('name and creator id required')
+			}
+			const timeStamp = new Date();
+			console.log(timeStamp)
+			const rooms = await dbClient.getCollection("chatDB", "rooms");
+			const room = await rooms.insertOne({name, created_by: userId, created_at: timeStamp});
+			if (!room.insertedId) {
+				throw new Error('failed to store chat room');
+			}
+			const newChat = {
```

Comment:
Why are we having two different models for same collection (chats collection)?? checkout this one you created and the other one in getChat function